src/elements/common/annotator-context/withAnnotations.tsx

```diff
@@ -129,16 +131,40 @@ export default function withAnnotations<P extends object>(
             }
         };
 
+        handleAnnotationsFetchError = ({ error }: { error: Error }) => {
+            this.fetchError = error;
+        };
+
         handleAnnotator = (annotator: Annotator): void => {
             this.annotator = annotator;
             this.annotator.addListener('annotatorevent', this.handleAnnotatorEvent);
+
+            if (this.fetchError) {
+                this.handleSidebarFetchError(this.fetchError);
+            }
+        };
+
+        handleAnnotatorCreate = (annotator: Annotator): void => {
+            annotator.addListener('annotations_fetch_error', this.handleAnnotationsFetchError);
+        };
+
+        handleSidebarFetchError = (error: Error) => {
+            const { annotator } = this;
+
+            if (!annotator) {
+                this.fetchError = error;
+                return;
+            }
+
+            annotator.emit('annotations_visible_set', false);
```

Comment:
We agreed that this edge case was unlikely and that handling it would add unnecessary complexity. For example, what happens if the fetch succeeds later?

Reply:
We need a strategy to solve the product requirement for not showing annotations in either the content or sidebar if either one of the fetch calls fail, though, unless the requirement is able to change.